grails-app/services/au/org/ala/bie/ImportService.groovy

```diff
@@ -497,32 +497,29 @@ class ImportService implements GrailsConfigurationAware {
         }
 
         // get List of WordPress document URLs (each page's URL)
-        def pages = wordpressService.pages()
+        def pages = wordpressService.resources()
         def documentCount = 0
         def totalDocs = pages.size()
         def buffer = []
         log("WordPress pages found: ${totalDocs}") // update user via socket
 
         // slurp and build each SOLR doc (add to buffer)
-        pages.each { pageUrl ->
+        pages.any { pageUrl ->
```

Comment:
Not understanding why this has changed from each to any, since this will cause termination on the first match?

Reply:
I needed to terminate the loop and read that `any` allows termination via the return false (for testing as full scrape was slow, so wanted to just test 10 results) but `each` does not. I willl double check to see if the `each` behaves the same way.